wordpress-mobile/MediaPicker-iOS
# Update project to be  iOS 13 minimum version

This PR updates the podspec to request iOS 13 as a minimum.
It also updates the code to reflect that change and removes any deprecation warnings.

One of the most relevant deprecation was the removal of support of ForceTouch  preview and adopt long press for all devices.

To test:
 - Start the demo app
 - Open the media picker
 - Long Press on asset and check that preview works correctly
